### PR TITLE
Add more instructions now nextflow run.

### PR DESCRIPTION
**Pull request by mberacochea**
I added the -latest flag for the nextflow run command and included additional documentation on running the pipeline.

### CODE REVIEW

File: README.md
Comment (tgurbich) [inline README.md:1]: should this have 1 dash? `-latest`?

File: README.md [hunk 1]
```diff
@@ -302,7 +314,7 @@ To run the pipeline using a test dataset, execute the following command:
 ```bash
 wget https://raw.githubusercontent.com/EBI-Metagenomics/mettannotator/master/tests/test.csv
 
-nextflow run ebi-metagenomics/mettannotator \
+nextflow run --latest ebi-metagenomics/mettannotator \
```
Comment (tgurbich) [inline README.md:317]: should this have 1 dash? `-latest`?

**Comment by tgurbich** [comment]
Other than that, looks great, thanks a lot @mberacochea!

File: README.md (hunk 1 above)
Comment (mberacochea) [reply README.md:317]: Indeed!

**Pull request merged**